### PR TITLE
Remove kubeless and lambda mentions in docs

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:

- Remove kubeless and lambda mentions in docs:
  - remove kubeless examples and replace to `functions.serverless.kyma-project.io` examples
  - replace `Lambda/lambda` to `Function/function`
  - update outdated tutorials and point in some places to tutorials in serverless topic - removing repetitive tutorials/steps.

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/7737

### CODE REVIEW

File: docs/compass/08-01-basic-flow-tutorial.md [hunk 1]
```diff
@@ -52,33 +52,23 @@ For simplicity reasons, use the available Order Service as the sample external A
...
-
-    f. Click the **Functions** tab and wait until the function status is completed and marked as `RUNNING`.
+5. Bind to your function ServiceInstance by [this](/components/serverless/#tutorials-bind-a-service-instance-to-a-function) tutorial. Follow the instructions in the `Console UI` tab.
 
 ### Cleanup
 
 Clean up your cluster after going through this tutorial. To do so, delete your resources in the following order:
 
-1. Go to the **Functions** tab, unfold the vertical option menu and delete your function.
+1. Go to the **Functions** tab and delete `call-order-service` function.
```
Comment (majakurcius) [inline docs/compass/08-01-basic-flow-tutorial.md:65]:
```suggestion
1. Go to the **Functions** tab and delete the `call-order-service` function.
```

File: docs/console/03-02-console-backend-service.md [hunk 2]
```diff
@@ -20,7 +20,8 @@ These are the available Console Backend Service pluggable modules which contain
 - `apigateway` relates to the API Gateway.
 - `authentication` relates to IDP Presets.
 - `application` relates to the Application Connector.
-- `kubeless` relates to Serverless.
+- `serverless` relates to Serverless.
+- `eventing` relates to Knative Eventing.
 - `rafter` relates to Rafter.
 - `servicecatalog` relates to the Service Catalog, including Service Classes, Service Instances, and Service Bindings.
```
Comment (majakurcius) [inline docs/console/03-02-console-backend-service.md:26]:
```suggestion
- `servicecatalog` relates to the Service Catalog, including ServiceClasses, ServiceInstances, and ServiceBindings.
```

Comment (magicmatatjahu) [reply docs/console/03-02-console-backend-service.md:26]: In other tutorials we have CR written separately and so I prefer to leave the name separated

File: docs/compass/08-01-basic-flow-tutorial.md [hunk 3]
```diff
@@ -52,33 +52,23 @@ For simplicity reasons, use the available Order Service as the sample external A
 
 2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package and clicking the **Add** button in the top-right corner of the page.
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply docs/compass/08-01-basic-flow-tutorial.md:53]: In other tutorials we have CR written separately and so I prefer to leave the name separated :)

File: docs/compass/08-01-basic-flow-tutorial.md [hunk 4]
```diff
@@ -50,35 +50,25 @@ For simplicity reasons, use the available Order Service as the sample external A
 
 1. Go back to the Kyma Console UI. You can see that the `test-app` Application is registered in the **Applications** view. Select `test-app` and bind it to your Namespace by selecting the **Create Binding** button.
 
-2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package and clicking the **Add** button in the top-right corner of the page.
+2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the Service Instance by choosing your Package and clicking the **Add** button in the top-right corner of the page.
```
Comment (majakurcius) [inline docs/compass/08-01-basic-flow-tutorial.md:53]:
```suggestion
2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the ServiceInstance by choosing your Package and clicking the **Add** button in the top-right corner of the page.
```

I did mean that lack of space in `ServiceInstance` 😄 Please see the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#capitalized-terms).

File: docs/compass/08-01-basic-flow-tutorial.md [hunk 5]
```diff
@@ -50,35 +50,25 @@ For simplicity reasons, use the available Order Service as the sample external A
...
-    b. In the **Settings & Code** section, click the **Select Function Trigger** button and expose your function via HTTPS.
-
-    c. Untick the **Enable authentication** field as there is no need to secure the connection for the purpose of this tutorial. Click **Add**.
-
-    d. Scroll down to the end of your function view and bind your function to your instance by clicking the **Create Service Bindings** button in the **Service Bindings** section. Choose the ServiceInstance you want to bind your function to, and click **Create Service Bindings**.
-
-    e. Save the settings in the right top-right corner of the page.
-
-    f. Click the **Functions** tab and wait until the function status is completed and marked as `RUNNING`.
+5. Bind the Service Instance to your function following [this](/components/serverless/#tutorials-bind-a-service-instance-to-a-function) tutorial. Follow the instructions in the `Console UI` tab.
```
Comment (majakurcius) [inline docs/compass/08-01-basic-flow-tutorial.md:59]:
```suggestion
5. Bind the ServiceInstance to your function following [this](/components/serverless/#tutorials-bind-a-service-instance-to-a-function) tutorial. Follow the instructions in the `Console UI` tab.
```

I did mean that lack of space in `ServiceInstance` 😄 Please see the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#capitalized-terms).

File: docs/console/03-02-console-backend-service.md (hunk 2 above)
Comment (majakurcius) [reply docs/console/03-02-console-backend-service.md:26]: I also hesitated about that as it's true we have inconsistencies in our documentation, but I talked about it with other TWs and we decided to stick to the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#capitalized-terms). 
We'll have a follow-up task to unify it in our docs.

**Pull request merged**